packages/lib/shared/services/api/apollo.client.ts

```diff
@@ -18,7 +18,6 @@ export function createApolloClient() {
   })
 
   return new ApolloClient({
-    ssrMode: typeof window === 'undefined',
```

Comment:
why is this no longer needed?

Reply:
https://github.com/apollographql/apollo-client-nextjs/pull/358

it's a bit harder to find out *why* the option was removed though

Reply:
i guess it was never needed? https://github.com/apollographql/apollo-client-nextjs/issues/350